recipes/_update_packages.rb

```diff
@@ -26,3 +26,45 @@
     command "DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::=\"--force-confdef\" -o Dpkg::Options::=\"--force-confold\" upgrade && apt-get autoremove"
   end
 end
+
+# Reboot after packages update
+if !tagged?('rebooted') && node['cfncluster']['update_packages']['reboot'] == 'true'
+  # On Ubuntu, the /tmp folder is erased by default after a reboot and its lifecycle is managed
+  # by different tools, depending on the Ubuntu version
+  if node['platform'] == 'ubuntu'
+    if node['platform_version'] == "16.04"
+      file '/etc/tmpfiles.d/tmp.conf' do
+        content 'd /tmp 1777 root root 1d'
+      end
+    end
+    if node['platform_version'] == "14.04"
+      execute 'sed' do
+        command "sed -i s/#TMPTIME=0/TMPTIME=1/g /etc/default/rcS"
```

Comment:
What does this do?

Reply:
It prevents the /tmp folder to be erased after reboot, which is the default on Ubuntu. /tmp folder is where all the kitchen test environment is copied.
http://manpages.ubuntu.com/manpages/trusty/man5/rcS.5.html